docs/deprecated.md

```diff
@@ -19,6 +19,19 @@ The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
+### Pushing and pulling with image manifest v2 schema 1
+
+**Deprecated in Release: v19.03.0**
+**Will be removed in Release: v19.09.0**
+
+The image manifest
+[v2 schema 1](https://github.com/docker/distribution/blob/fda42e5ef908bdba722d435ff1f330d40dfcd56c/docs/spec/manifest-v2-1.md)
+format is deprecated in favor of the
+[OCI distribution spec](https://github.com/opencontainers/distribution-spec/blob/master/spec.md)
```

Comment:
should we pin this to a commit as well?

Reply:
pinned to https://github.com/opencontainers/distribution-spec/blob/v1.0.0-rc0/spec.md